Worlds/FelipeWorld/1.tscn

```diff
@@ -0,0 +1,8 @@
+[gd_scene load_steps=2 format=4 uid="uid://c1eogjmefwg4m"]
+
+[ext_resource type="TileSet" uid="uid://caem805hxljgn" path="res://CandyWrapper/TileSet.tres" id="1_l4q0s"]
+
+[node name="TileMapLayer" type="TileMapLayer"]
+use_parent_material = true
+tile_map_data = PackedByteArray("AAD/////AwAAAAAAAAD//wAAAwAAAAAAAAD//wEAAwAAAAAAAAD//wIAAwAAAAAAAAD//wMAAwAAAAAAAAD//wQAAwAAAAAAAAD//wUAAwAAAAAAAAD//wYAAwAAAAAAAAD//wcAAwAAAAAAAAD//wgAAwAAAAAAAAD//wkAAwAAAAAAAAD//wwAAwAAAAAAAAD//w0AAwAAAAAAAAD//w4AAwAAAAAAAAD//w8AAwAAAAAAAAD//xAAAwAAAAAAAAD//xEAAwAAAAAAAAD//xIAAwAAAAAAAAAAAP//AwAAAAAAAAAAABIAAwAAAAAAAAABAP//AwAAAAAAAAABABIAAwAAAAAAAAACAP//AwAAAAAAAAACABIAAwAAAAAAAAADAP//AwAAAAAAAAADABIAAwAAAAAAAAAEAP//AwAAAAAAAAAEABIAAwAAAAAAAAAFAP//AwAAAAAAAAAFABIAAwAAAAAAAAAGAP//AwAAAAAAAAAGABIAAwAAAAAAAAAHAP//AwAAAAAAAAAHABIAAwAAAAAAAAAIAP//AwAAAAAAAAAIABIAAwAAAAAAAAAJAP//AwAAAAAAAAAJABIAAwAAAAAAAAAKAP//AwAAAAAAAAAKABIAAwAAAAAAAAALAP//AwAAAAAAAAALABIAAwAAAAAAAAAMAP//AwAAAAAAAAAMABIAAwAAAAAAAAANAP//AwAAAAAAAAANABIAAwAAAAAAAAAOAP//AwAAAAAAAAAOABIAAwAAAAAAAAAPAP//AwAAAAAAAAAPABIAAwAAAAAAAAAQAP//AwAAAAAAAAAQABIAAwAAAAAAAAARAP//AwAAAAAAAAARABIAAwAAAAAAAAASAP//AwAAAAAAAAASAAAAAwAAAAAAAAASAAEAAwAAAAAAAAASAAIAAwAAAAAAAAASAAMAAwAAAAAAAAASAAQAAwAAAAAAAAASAAUAAwAAAAAAAAASAAYAAwAAAAAAAAASAAcAAwAAAAAAAAASAAgAAwAAAAAAAAASAAkAAwAAAAAAAAASABIAAwAAAAAAAAD//woAAwAAAAAAAAD//wsAAwAAAAAAAAASAAoAAwAAAAAAAAAIAAoAAQAAAAAAAAASAAsAAwAAAAAAAAASAAwAAwAAAAAAAAASAA0AAwAAAAAAAAASAA4AAwAAAAAAAAASAA8AAwAAAAAAAAASABAAAwAAAAAAAAASABEAAwAAAAAAAAABAAEAAAABAAAAAAACAAEAAAABAAAAAAAAAAEAAAABAAAAAAAAAAIAAAABAAAAAAAAAAMAAAABAAAAAAAAAAQAAAABAAAAAAAAAAUAAAABAAAAAAAAAAYAAAABAAAAAAABAAMAAAABAAAAAAAEAAEAAAABAAAAAAAEAAIAAAABAAAAAAAEAAMAAAABAAAAAAAEAAQAAAABAAAAAAAEAAUAAAABAAAAAAAEAAYAAAABAAAAAAAFAAYAAAABAAAAAAAFAAMAAAABAAAAAAAFAAEAAAABAAAAAAAHAAEAAAABAAAAAAAHAAIAAAABAAAAAAAHAAMAAAABAAAAAAAHAAQAAAABAAAAAAAHAAUAAAABAAAAAAAHAAYAAAABAAAAAAAIAAYAAAABAAAAAAAKAAEAAAABAAAAAAAKAAIAAAABAAAAAAAKAAMAAAABAAAAAAAKAAQAAAABAAAAAAAKAAUAAAABAAAAAAAKAAYAAAABAAAAAAAMAAEAAAABAAAAAAAMAAIAAAABAAAAAAAMAAMAAAABAAAAAAAMAAQAAAABAAAAAAAMAAUAAAABAAAAAAAMAAYAAAABAAAAAAANAAEAAAABAAAAAAAOAAIAAAABAAAAAAANAAMAAAABAAAAAAAQAAEAAAABAAAAAAAQAAIAAAABAAAAAAAQAAMAAAABAAAAAAAQAAQAAAABAAAAAAAQAAUAAAABAAAAAAAQAAYAAAABAAAAAAARAAYAAAABAAAAAAARAAMAAAABAAAAAAARAAEAAAABAAAAAAAHAAoAAAABAAAAAAAGAAoAAAABAAAAAAAGAAkAAAABAAAAAAAFAAkAAAABAAAAAAAEAAkAAAABAAAAAAAEAAoAAAABAAAAAAAEAAsAAAABAAAAAAAEAAwAAAABAAAAAAAFAAwAAAABAAAAAAAFAA0AAAABAAAAAAAGAA0AAAABAAAAAAAHAA4AAAABAAAAAAAIAA4AAAABAAAAAAAIAA8AAAABAAAAAAAJAA8AAAABAAAAAAAJAAoAAAABAAAAAAAKAAoAAAABAAAAAAAKAAkAAAABAAAAAAALAAkAAAABAAAAAAAMAAkAAAABAAAAAAAMAAoAAAABAAAAAAAMAAsAAAABAAAAAAAMAAwAAAABAAAAAAALAAwAAAABAAAAAAALAA0AAAABAAAAAAAKAA0AAAABAAAAAAAKAA4AAAABAAAAAAAJAA4AAAABAAAAAAAFAAoAAAABAAAAAAAFAAsAAAABAAAAAAALAAoAAAABAAAAAAALAAsAAAABAAAAAAADAAoAAAABAAAAAAANAAoAAAABAAAAAAADAAsAAAABAAAAAAANAAsAAAABAAAAAAAIAA0AAAABAAAAAAAFAAAAAgAAAAAAAAAQAAAAAgAAAAAAAAAGAA4AAAAAAAAAAAAHAA8AAAAAAAAAAAAIABAAAAAAAAAAAAA=")
```

Comment:
I found it particularly challenging to stomp on the goober at the top-left/top-right of this map!

![image](https://github.com/user-attachments/assets/b3de6dee-bfa1-4fad-bacf-3c2470867143)